process-engine/iam
# 🔊 Increase log verbosity

## Changes

Add identity and claim to error logs, when a claim check fails.

## Issues

Part of https://github.com/process-engine/process_engine_runtime/issues/475

PR: #16

## How to test the changes

- Make a claim check you are sure will fail
- See that the error will contain the identity that made the request, as well as the claim that was checked
